src/Psalm/Internal/CliUtils.php

```diff
@@ -622,4 +623,25 @@ public static function convertMemoryLimitToBytes(string $limit): int
 
         return (int)$limit;
     }
+
+    public static function initPhpVersion(array $options, Config $config, ProjectAnalyzer $project_analyzer): void
+    {
+        $source = null;
+
+        if (isset($options['php-version'])) {
+            if (!is_string($options['php-version'])) {
+                die('Expecting a version number in the format x.y' . PHP_EOL);
```

Comment:
This check can be dropped here, it seems to be checked again in project analyzer below

Reply:
It is to avoid MixedAssignment to `$version` variable.